LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, Simone Basso, Arturo Filastò, The Tor Project, Inc.
```

Comment:
The heading is wrong. It should say:

```
Copyright 2017 Open Observatory of Network Interference (OONI), The Tor Project
```

You should copy paste this file:
https://github.com/OpenObservatory/legal/blob/master/LICENSE-BSD-3-Clause.txt